partcad-cli/src/partcad_cli/click/commands/list/parts.py

```
@@ -23,20 +23,27 @@
 )
 @click.argument("package", type=str, required=False, default=".")  # help='Package to retrieve the object from'
 @click.pass_obj
-def cli(ctx, used_by, recursive, package):
+def cli(ctx: Context, used_by: str | None, recursive: bool, package: str):
```

Ruff (0.8.2)

`cli` is too complex (14 > 10)

(C901)